spec/model/model_spec.cr

```diff
@@ -72,11 +72,11 @@ module ModelSpec
     end
   end
 
-  Clear::Migration::Manager.instance.reinit!
-  ModelSpecMigration123.new.apply(Clear::Migration::Direction::UP)
+  temporary do
+    Clear::Migration::Manager.instance.reinit!
+    ModelSpecMigration123.new.apply(Clear::Migration::Direction::UP)
```

Comment:
Found a different bug when running the specs, where the user table created by ModelSpecMigration123 was causing the "users" table to linger between specs.